cli/copr_cli/main.py

```diff
@@ -590,6 +590,18 @@ def action_mock_config(self, args):
         build_config.rootdir = "{0}-{1}_{2}".format(ownername.replace("@", "group_"), projectname, args.chroot)
         print(MockProfile(build_config))
 
+    @staticmethod
+    def _print_project_info(project):
+        print(f"Name: {project.name}")
```

Comment:
There are no f-strings on EPEL7, so this is what makes the CI fail.
You'll need to use `.format`

Reply:
+1, we want to keep EPEL7 compatibility in `python-copr` and `copr-cli`.